### ISSUE TITLE
Event Survey

### ISSUE THREAD

**Issue by Rdhivakar** [Attendize/Attendize#541]
I have 2 issue here. 
1. Not able to view the survey created for the particular ticket. Example. For the tickets sold,  I want to know whether they are vegetarian or non-vegetarian in order to arrange food. For this purpose, I created an event survey with the required options. I am not able to see this option while booking the ticket in the event page.
2. When an order is cancelled,  Status column in the event order page is not updating as cancelled. It is still displayed as completed. Whereas the original order status is cancelled.

**Comment by philip-loyd** [comment]
Yes, the survey questions should show on the event page. I don't see a reason why they should be on a separate page. Great feature request.
